src/pages/dar_application/ProgressReportApplication.tsx

```diff
@@ -19,10 +29,23 @@ export const ProgressReportApplication = ({dar, readOnlyMode=true}: ProgressRepo
                     <h4>Intellectual Property Summary</h4>
                     {dar?.intellectualPropertySummary ?? "PLACEHOLDER Intellectual Property Summary"}
                 </div>
-                {dar?.parentId && <div>
+                <div data-cy='remove-datasets'>
+                    <div className='progress-report-step-card'>
+                        <h2>Step 3: Dataset(s) in this DAR</h2>
+                        <p style={{ marginBottom: '1rem' }}>Currently selected datasets:</p>
+                        <SelectableDatasets
+                            disabled={readOnlyMode}
+                            datasets={datasets}
+                            setSelectedDatasets={(selectedDatasets) => {
+                                setFormState({...formState, datasetIds: selectedDatasets.map((ds) => ds.datasetId)})}
```

Comment:
I think this should load the initial form state as the datasets, and setFormState should modify the list of datasets.